Description: Fix copy method of GnuPG::Options.
 The result of not checking for definedness here is to never copy the
 meta_immutable value successfully (as that is true by default).  This led
 to a FTBFS when running non-interactively.
Author: Tim Retout <tim@retout.co.uk>
Bug-Debian: http://bugs.debian.org/549743

--- a/lib/GnuPG/Options.pm
+++ b/lib/GnuPG/Options.pm
@@ -97,7 +97,7 @@
 
     foreach my $field ( BOOLEANS, SCALARS, LISTS ) {
         my $value = $self->$field();
-        next unless $value;
+        next unless defined $value;
         $new->$field($value);
     }
 
